style.css

```diff
@@ -34,12 +34,43 @@ main {
   width: 100%;
 }
 
+ul {
+  list-style-type: none;
+}
+
 #toolbar nav ul {
   display: flex;
   justify-content: space-between;
   list-style-type: none;
 }
 
+.programming-lang li {
+  display: flex;
+  align-items: center;
+  background-color: #f7f7f9;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  color: #253858;
+  margin: 16px 20px 16px 0;
+  gap: 16px;
+  padding: 10px;
+}
+
+main .card-works .card .card-description .web-languages li {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin: 10px 0;
+}
```

Comment:
- [ ] Great work with the design so far. However, you still have some little things to do before you go. 💪 Kindly, and as per project requirements, use both flexbox and grid CSS features in order to wrap up the `Work` section in your portfolio. Refresh your memory [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20in%20%22Works%22%20section%20you%20should%20use%20CSS%20Grid%20and%20Flexbox%20combined%20(it%20is%20mandatory%20that%20you%20use%20both).%20Use%20CSS%20Grid%20in%20the%20main%20container%20that%20contains%20all%20of%20the%20cards%20and%20use%20Flexbox%20to%20lay%20out%20an%20individual%20card.). 🙏